templates/sms.1.txt

```diff
@@ -0,0 +1 @@
+{{t "As requested, here is a link to install Firefox on your mobile device: %(link)s"}}
```

Comment:
> I wonder if we should bother with submitting this for translation while we still test this EN locales.

Fwiw, I disagree. What is the benefit of not localizing?

> ...if we use short-codes, we might have to store already translated strings with the SMS provider.

I created a separate issue to discuss this quetion: mozilla/fxa-auth-server#1645.